week-1-practise/level-1/8-function-output-1.js

```diff
@@ -1,12 +1,16 @@
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
 
 var result = getNumber();
 console.log(result);
 
 // Answer these questions:
 // - What does this program do?
+// Math.random() returns a random number in the range 0 and 10.
```

Comment:
Check out the MDN (https://developer.mozilla.org/en-US/) page for `Math.random` - does it really return a value that would allow 10 to be the result of this function?